CHANGES.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## v0.9.0
+
+Users willing to use IMPIComm, IOCStats, pyDevSup or SNMP modules should update
```

Comment:
The official [iocStats repo](https://github.com/epics-modules/iocStats) writes the name like this. What do you think?
```suggestion
Users willing to use IMPIComm, iocStats, pyDevSup or SNMP modules should update
```

Same in https://github.com/cnpem/epics-in-docker/blob/3d27ac658a1a9062f5314198ee12afafbdec2b4b/CHANGES.md?plain=1#L19 (my bad, sorry)

Reply:
> What do you think?

Let's follow it. Thanks!